dependencies.gradle

```diff
@@ -20,6 +20,7 @@ project.ext.versions = [
     jsonassert:'1.3.0',
     jackson_core:'2.8.8',
     jackson_databind:'2.8.8.1',
+    jackson_dataformat_yaml:'2.8.8',    
```

Comment:
Any transitive dependencies that are brought in that could conflict with existing ones?

Reply:
Good question.  I think this is safe though.

As far as I can tell this depends on Jackson core (which we're already using) and SnakeYAML which we're not using anywhere else (so no version conflict) and has no additional dependencies.

Reply:
We will have to watch the version of Jackson core vs. the one that comes in Spark, for atlas-generator.... Thanks.